foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec

```diff
@@ -0,0 +1,55 @@
+%define _binaries_in_noarch_packages_terminate_build 0
+
+Name: foreman-bootloaders-redhat
+Version: 201705231433
+Release: 1%{?dist}
+Summary: Binaries of Grub1, Grub2 and Shim bootloaders
+
+Group: Applications/System
+License: GPLv2+ and GPLv3+ and BSD
+URL: https://github.com/theforeman/foreman-bootloaders
+Source0: http://downloads.theforeman.org/foreman-bootloaders/%{name}-%{version}.tar.bz2
+BuildArch: noarch
+
+Requires: /var/lib/tftpboot
```

Comment:
It's not a matter of right or wrong, it's completely configurable on the proxy and xinetd, the point is that the package should respect the existing configuration (otherwise it'd just not work, like on my installation). The default is the same in all operating systems AFAIK. Here are my config files:

https://gist.github.com/dLobatog/bd32aba094a91bf8b914a7f148d5db82